src/Tests/Microsoft.NET.Build.Containers.IntegrationTests/CreateNewImageTests.cs

```diff
@@ -46,28 +48,33 @@ public void CreateNewImage_Baseline()
             .Execute()
             .Should().Pass();
 
-        CreateNewImage task = new CreateNewImage();
+        CreateNewImage task = new();
+
+        (IBuildEngine buildEngine, List<string?> errors) = SetupBuildEngine();
```

Comment:
I love this - what about a `T SetupBuildEngine<T>()` that would also create the task and do the `.BuildEngine` setter before returning, so that it's harder to mess up the construction?